core/src/jmh/java/org/apache/iceberg/AppendBenchmark.java

```diff
@@ -86,6 +86,7 @@ public class AppendBenchmark {
 
   @Setup
   public void setupBenchmark() {
+    dropTable();
```

Comment:
isn't cleanup already called in line 96 in the tearDown method? in the other comment, you mentioned that table exist exception, is it leftover from previous run that was interrupted and not cleaned up?

Reply:
I honestly didn't check, I think it was probably the previous run OOMing